setup.cfg

```diff
@@ -2,7 +2,7 @@
 universal = 1
 
 [tool:pytest]
-addopts = --cov-report= --cov=devpi_builder tests
+;addopts = --cov-report= --cov=devpi_builder tests
```

Comment:
As far as I can see this breaks the coverage analysis in the Travis build. I assume you changed this to be able to run without coverage locally. Please either re-add or explicitly add the arguments to the test invocation in the Travis build.

FYI: You can also invoke py.test with `--no-cov` if you locally want to run without coverage.